docs/install/operator/configuring-serving-cr.md

```diff
@@ -1,24 +1,12 @@
 # Configuring the Knative Serving Operator custom resource
 
-You can configure Knative Serving with the following options:
-
-- [Version configuration](#version-configuration)
```

Comment:
I think this is helpful instead of having to scroll down no (although contents are shown on the right tab)?
If we remove this we should remove the Eventing one too for UX consistency.